packages/kokoas-server/src/handleRequest/putInvoiceReport/parseInvoiceDat.test.ts

```diff
@@ -0,0 +1,13 @@
+import { getInvoiceById } from 'api-kintone/src/invoice/getInvoiceById';
+import { parseInvoiceDat } from './parseInvoiceDat';
+
+describe('parsing invoice record', () => {
+  it('should validate file', async () => {
```

Comment:
テストしている関数を見る限り、以下はいかがでしょうか。

```suggestion
  it('returns a parsed invoice record object', async () => {
```

Reply:
ご指摘ありがとうございます。
英語がまだまだ難しいので、とても助かりました。
[0717b84](https://github.com/Lorenzras/yumecoco-monorepo/pull/157/commits/0717b84476b6ce63d556d6570e916b4f7c771b66)にて修正しました。
ご確認をお願いいたします。